test/dk/simongray/test/ml_test.clj

```diff
@@ -2,15 +2,26 @@
   (:require [clojure.test :refer :all]
             [tech.v3.dataset.modelling :as ds-mod]
             [tech.v3.dataset :as ds]
-            [tech.v3.ml :as ml]
+            [scicloj.metamorph.ml :as ml]
             [dk.simongray.datalinguist.ml.crf :refer :all]))
 
-(deftest train-predict
-  (let [train-ds   (-> (ds/->dataset "./test/data/jane-austen-emma-ch1.tsv" {:header-row? false :key-fn keyword})
-                       (ds-mod/set-inference-target :column-1))
-        model      (ml/train train-ds {:model-type :corenlp/crf})
+(def train-ds
+  (-> (ds/->dataset "./test/data/jane-austen-emma-ch1.tsv" {:header-row? false :key-fn keyword})
+      (ds-mod/set-inference-target :column-1)))
+
+(deftest train-predict-default
+  (let [model      (ml/train train-ds {:model-type :corenlp/crf})
         text-ds    (ds/->dataset {:column-0 ["I like Mr. X and I like  Mr. Y" "Smith" "Mrs." "Weston"]})
         prediction (ml/predict text-ds model)]
     (is (= [["PERS" 7 10]
             ["PERS" 25 30]]
            (first (:ner prediction))))))
+
+(deftest train-predict-defaul-no-ngrams
+  (let [model      (ml/train train-ds {:model-type :corenlp/crf
+                                      :useNGrams "false"})
```

Comment:
Model options would be specified like this

Reply:
Yup, let's go with that for now. The more I think about it, the more I think I prefer keeping the default case from CoreNLP. Ideally people should be able to copy-paste names they find in CoreNLP documentation.